### PR TITLE
Add Keywords To Setup Metadata

### PR DESCRIPTION
**Pull request by ClaytonPassmore**
This will (hopefully) allow extensions to be more easily discovered.

See https://github.com/jupyterlab/jupyterlab/issues/1113

### CODE REVIEW

File: {{cookiecutter.extension_name}}/setup.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class NodeModulesMissing(Exception):
     packages             = ['{{ cookiecutter.extension_name }}'],
     author               = '{{ cookiecutter.author_name }}',
     author_email         = '{{ cookiecutter.author_email }}',
+    keywords             = 'jupyterlab extension',
```
Comment (willingc) [inline {{cookiecutter.extension_name}}/setup.py]: @ClaytonPassmore Nice! What do you think about adding the individual words to the list? 

```
['jupyterlab extension`, `jupyterlab`, `extension`]
```

Comment (jasongrout) [reply {{cookiecutter.extension_name}}/setup.py]: I would prefer having at least `jupyterlab` also by itself - maybe 'jupyterlab' and 'jupyterlab extension'?

Comment (ClaytonPassmore) [reply {{cookiecutter.extension_name}}/setup.py]: Sure thing!

**Comment by jasongrout** [comment]
Looks good to me. Thanks!

**Comment by willingc** [comment]
Thanks @ClaytonPassmore. Looks good!

**Comment by ClaytonPassmore** [comment]
No problem! :smile:

**Pull request merged**